apps/bare-expo/babel.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(api) {
   const moduleResolverConfig = {
     alias: {
       '~expo': 'expo',
-      expo: require.resolve('./moduleResolvers/expoResolver.js'),
+      // expo: require.resolve('./moduleResolvers/expoResolver.js'),
```

Comment:
Is this change intentional? On the other hand, I suppose we can already remove `expoResolver.js` 🤔

Reply:
yes it's intentional because without it the tests fail because the entry point doesn't work 2b115a0